### ISSUE TITLE
Feature Request: Consider avoiding the Google API

### ISSUE THREAD

**Issue by sunjam** [IamTheFij/google-photo-to-vcard#1]
[Hi, I think there is a sincere interest in avoiding Google API's while introducing facial recognition in Nextcloud/owncloud/vCard.](https://help.nextcloud.com/t/ownphotos-now-offers-experimental-cardav-facial-recognition/29487)  Please check out [ownPhotos, which afaik uses a Python script for facial recognition that does not phone home to google while recreating the Google Photos experience.](https://github.com/hooram/ownphotos-backend/)

Perhaps you could offer a similar experience with your own coding!

**Comment by IamTheFij** [comment]
This is already considered. :)

This is not a web application, but instead a migration script. My contacts all already had photos when I was still using Google. When I migrated their contacts to Nextcloud, the photos didn't migrate with them. (😢) So I wrote this script and now my Nextcloud contacts all have photos!

You never again have to run this script if you're ready to maintain your contact photos in Nextcloud.
